docs/plugins/plugin-script.asciidoc

```diff
@@ -64,13 +64,35 @@ sudo bin/elasticsearch-plugin install [url] <1>
 -----------------------------------
 <1> must be a valid URL, the plugin name is determined from its descriptor.
 
-For instance, to install a plugin from your local file system, you could run:
+==== Unix File System
+
+To install a plugin from your local file system at `/path/to/plugin.zip`, you could run:
 
 [source,shell]
 -----------------------------------
 sudo bin/elasticsearch-plugin install file:///path/to/plugin.zip
 -----------------------------------
 
+==== Windows File System
+
+On Windows, the `file` path is much easier to work with using absolute paths only. Absolute paths
```

Comment:
Looks like we recommend using an absolute path for unix file systems as well. Maybe drop the mention of absolute paths entirely?